emcc.py

```diff
@@ -1886,21 +1900,23 @@ def do_minify(): # minifies the code. this is also when we do certain optimizati
            '--mapFileBaseName', target,
            '--offset', str(0)])
 
-    # Move final output to the js target
-    shutil.move(final, js_target)
```

Comment:
Yeah, the intent was here to optimize away a copy to the deployed location, just to read that file back in to change the line endings. Would this be good if renamed to `line_endings.move_and_change_line_endings`? (while allowing src and dest to be the same file, in which case it would do the line ending change in place on the file)